server/src/test/java/org/elasticsearch/action/bulk/BulkProcessorRetryIT.java

```diff
@@ -55,7 +55,7 @@ protected Settings nodeSettings(int nodeOrdinal) {
                 //.put("thread_pool.listener.queue_size", 1)
                 .put("thread_pool.get.queue_size", 1)
                 // default is 50
```

Comment:
Nitpick, but can we update this to 200 too (or remove the comment?).

Reply:
I pushed f420edbc32ebcba0d47d3a9b119cc0c63b43c854.